samples/Commerce/Commerce.UI/Styles/FeedView.xaml

```diff
@@ -37,7 +37,7 @@
 				<DataTemplate>
 					<not_skia:Grid>
 						<winui:ProgressRing
-							Style="{StaticResource MaterialProgressRingStyle}"
+							Style="{StaticResource ProgressRingStyle}"
```

Comment:
This is good for now but we should make a pass afterwards to remove the Style setters for the styles that are implicit by default

Reply:
My goal is to do the migration first and check for those details afterwards.